site-structure.json

```diff
@@ -39,14 +39,25 @@
                 [
                     { "pageName": "Events", "href": "/Events.html" },
                     { "pageName": "Infusion Event System", "href": "/InfusionEventSystem.html" },
+                    { "pageName": "Priorities", "href": "/Priorities.html" },
```

Comment:
Given that priorities apply to so much more than just events, I wonder if we should put this in a different section of the ToC - maybe the "Defining Components" section, since that's where most of the broadly-relevant stuff is.